src/main/kotlin/me/zeroeightsix/kami/util/VectorMath.kt

```diff
@@ -30,6 +30,9 @@ val Vec3i.asVec3d
 val BlockPos.asVec
     get() = Vec3d(this.x.toDouble(), this.y.toDouble(), this.z.toDouble())
```

Comment:
Looks like this is never used, maybe delete this?

Reply:
This wasn't added by @LordMZTE. It is used in Scaffold.

Reply:
oh ok